spec/unit/resource_controller_spec.rb

```diff
@@ -186,7 +186,7 @@ def call_after_destroy(obj); end
     end
   end
 
-  describe 'retreiving the resource collection' do
+  describe 'retrieving the resource collection' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.